code/model_infall.c

```diff
@@ -7,7 +7,7 @@
 #include "core_allvars.h"
 #include "core_proto.h"
 
-
+#define UNUSED(foo) (void)(foo)
```

Comment:
I wonder if we should move this to `core_allvars.h`. Or at least undefine `UNUSED` at the bottom of the file

Reply:
If it were shifted to `core_allvars.h`, then yes, I agree that undefining `UNUSED` at the end would make sense. On the other hand, `core/model_infall.c` is not going to be included in any other file, so I'm not convinced of the point of undefining it.